<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerInfoWidget</class>
 <widget class="QWidget" name="ServerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(48, 47, 47, 90%);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="serverLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 20px;
background: transparent;
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="closeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
    background-color: rgba(32,31,31, 75%);
    color: silver;
    border: 1px solid #3A3939;
    margin: 0px;
    padding: 0px;
    outline: none;
    border-radius: 3px;
    font-size: 20px;
}

QPushButton:disabled {
    background-color: rgba(48, 47, 47, 75%);
    border: none;
    margin: 0px;
    padding: 0px;
    /*border-radius: 3px;*/
    color: #454545;
}

QPushButton:checked {
    background-color: #78879b;
    border-radius: 3px;
}


QPushButton:hover {
border: 1px solid #78879b;
color: silver;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1,
    stop: 0 #302F2F, stop: 1 #484846);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qdarkstyle/style.qrc">
          <normaloff>:/qss_icons/rc/close.png</normaloff>:/qss_icons/rc/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qdarkstyle/style.qrc"/>
 </resources>
 <connections/>
</ui>
